src/API/Helpers/LocalizationHelper.cs

```diff
@@ -10,28 +10,37 @@ namespace SerousCommonLib.API.Helpers {
 	/// A helper class for using localized text
 	/// </summary>
 	public static class LocalizationHelper {
-		internal static readonly MethodInfo LocalizationLoader_AutoloadTranslations = typeof(LocalizationLoader).GetMethod("AutoloadTranslations", BindingFlags.NonPublic | BindingFlags.Static);
-		internal static readonly MethodInfo LocalizationLoader_SetLocalizedText = typeof(LocalizationLoader).GetMethod("SetLocalizedText", BindingFlags.NonPublic | BindingFlags.Static);
-		internal static readonly FieldInfo LanguageManager__localizedTexts = typeof(LanguageManager).GetField("_localizedTexts", BindingFlags.NonPublic | BindingFlags.Instance);
```

Comment:
these fields are still needed in `TML_2022_09` context

Reply:
Put the various things into the correct contexts